src/components/SortableTable/selectionStore.js

```diff
@@ -56,7 +56,7 @@ export const getters = {
     }
 
     // Go through all the selected items and add the actions (which were already identified above)
-    // as availalable for some (or all) of the selected nodes
+    // as available for some (or all) of the selected nodes
```

Comment:
This fix should be made in [dashboard](https://github.com/rancher/dashboard/blob/2345527a916cde3909a5ca0ceae3b7b36f593603/components/SortableTable/selectionStore.js#L59).